static/app/components/forms/jsonForm.tsx

```diff
@@ -123,7 +127,15 @@ class JsonForm extends Component<Props, State> {
       return null;
     }
 
-    return <FormPanel title={title} fields={fields} {...formPanelProps} />;
+    return (
+      <FormPanel
+        title={title}
+        fields={fields}
+        {...formPanelProps}
```

Comment:
Shouldn't this `formPanelProps` already contain the props to be passed to the form panel? Typescript is saying they aren't there because it's casting with `Pick` above I think?

Reply:
Hey @k-fish , so before title fields and now id and initially collapsed comes from formGroup props passed to renderForm because we passed a `forms` prop and not just `fields` to `JsonForm`. [Here](https://github.com/getsentry/sentry/pull/55692/files#diff-9259689032b54efc0d44b1d44a2be5fa972e0d6995481bbc088132ded6ebc252L165). This is different

Reply:
I think we can combine formGroup and formPanelProps.

Reply:
Ah I think I get it now, the initiallyCollapsed state needs to come from the form group itself since we are doing individual collapsing, the formPanel props are just the overarching parent props for initially collapsed. 

We can leave it how you had it previously as an extra param to the function, where you're just passing `initiallyCollapsed` through and it overrides the FormPanel parent prop. Just add a test to assert behaviour when the parent `initiallyCollapsed` is set along side individual form `initiallyCollapsed` as that's maybe one of the edge cases.